src/lib/audio/audio-util.js

```diff
@@ -0,0 +1,18 @@
+class AudioUtil {
+    static computeRMS (samples) {
```

Comment:
Noob question about the export syntax style. I see that we use an eslint rule that that "requires the use of function expressions instead of function declarations" http://eslint.org/docs/rules/func-style. 

That prevents doing this, which I see in some codebases
```javascript
export function computeRMS (...) { ... }
```

 Is the proper way in "util" type files with multiple named exports to export at the bottom like this?

```javascript
const computeRMS = function (...) { ... }

// possibly other util functions

export {
    computeRMS,
    // the other named utils
};
```